app/Http/Controllers/TeamController.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\TeamMember;
+use Illuminate\Http\Request;
+use App\AppUser;
+use App\Team;
+
+
+class TeamController extends Controller
+{
+    public function create_team(Request $request){
+        $error_response['status_code'] = '0';
+        $success_response['status_code'] = '1';
+
+        if(!$request->has('rollNo')){
+            $error_response['status']='Error : rollNo not received.';
+            return response($error_response);
+        }
+
+        if(!$request->has('team_name')){
+            $error_response['status']='Error : team_name not received.';
+            return response($error_response);
+        }
+
+        if(!$request->has('event_id')){
+            $error_response['status']='Error : event id not received.';
+            return response($error_response);
+        }
+
+        $id = AppUser::where('rollNo','=',$request->rollNo)->value('id');
+        $app_user = AppUser::find($id);
```

Comment:
Recommended : `$appUser = AppUser::where('rollNo',$request->rollNo)->first();` as `rollNo` is unique and this will eliminate querying the `id` in the above statement.

Reply:
done